### ISSUE TITLE
Launching an installed app does nothing

### ISSUE THREAD

**Issue by dep** [filips123/PWAsForFirefox#229]
I've followed the installation instructions and everything finished without incident.  I installed my first "app" (Google Chrome) however upon trying to launch the app with the toolbar button, either in the URL bar or within the extension pop-up, nothing happens.  I'm on an Intel MBPro, running latest Firefox.

**Comment by dep** [comment]
When I launch the app manually here is the result:

<img width="536" alt="image" src="https://user-images.githubusercontent.com/55112925/193574780-4f326b9d-ddc6-48ba-a53b-b57b750aac5b.png">

**Comment by filips123** [comment]
Please try to run the app with `open -Wa "Google Chat"` from Terminal and check if it provides any logs. Does it happen just with Google Chat or also with other web apps?

**Comment by AryaTwirl** [comment]
Hi, I also have the same issue, although it only happens on my Hackintosh, but works well on my M1 Air. I have reinstalled firefox and the extension, but it still occurs. Cannot get a log either from the terminal, the terminal says it cannot find the app.

**Comment by AryaTwirl** [comment]
Upon further inspection, somehow on my hackintosh, the PWA Firefox does not create the "launch" executable file.

**Comment by AryaTwirl** [comment]
Finally solved the issues. Turns out you have to install XCode command line tools, as when looking at the log, it tells you something about missing xcrun at the command tools directory.

The solution is just to open Terminal and type xcode-select --install.

**Comment by filips123** [comment]
How did you install PWAsForFirefox? Because I think Homebrew itself requires Xcode CLT and cargo-make also installs them when buildining, so I'm not sure why you didn't have them installed.

I will probably also make it more noticable that you need them installed.

**Comment by AryaTwirl** [comment]
> How did you install PWAsForFirefox? Because I think Homebrew itself requires Xcode CLT and cargo-make also installs them when buildining, so I'm not sure why you didn't have them installed.
> 
> I will probably also make it more noticable that you need them installed.

Thank you for the reply.

And yes, that's exactly what I thought.

When I first made the comment that I had the same issue, I assumed that Xcode was already installed (and I did have Homebrew at that time for other usage). Yet, when I saw the log and the complaint about xcrun missing, I just thought that, well, let's just install Xcode again. And it then runs fine.

Maybe somehow along the way between me installing PWA and installing the web app, the Xcode got uninstalled. Somehow. 

But yeah, other than installing xcode again, I can't pinpoint any other culprits.
